examples/gno.land/r/sys/validators/v2/gno.mod

```diff
@@ -1,12 +1,13 @@
-module gno.land/r/sys/validators
+module gno.land/r/sys/validators/v2
 
 require (
 	gno.land/p/demo/avl v0.0.0-latest
+	gno.land/p/demo/dao v0.0.0-latest
 	gno.land/p/demo/seqid v0.0.0-latest
 	gno.land/p/demo/testutils v0.0.0-latest
 	gno.land/p/demo/uassert v0.0.0-latest
 	gno.land/p/demo/ufmt v0.0.0-latest
-	gno.land/p/gov/proposal v0.0.0-latest
 	gno.land/p/nt/poa v0.0.0-latest
 	gno.land/p/sys/validators v0.0.0-latest
+	gno.land/r/gov/dao/v2 v0.0.0-latest
```

Comment:
r/sys/validators shouldn't depend on r/gov/dao/v2

Reply:
Resolved in:

[d98d780](https://github.com/gnolang/gno/pull/2581/commits/d98d78051c2c91044d0c3d87e83f9b08e5441d68)